Makefile

```diff
@@ -33,19 +33,20 @@ node_modules: package-lock.json
 tests: node_modules $(BUILDDIR)/$(JSON_ACIS) $(BUILDDIR)/$(BYTECODES)
 	npm test
 
-$(BUILDDIR)/js: | $(BUILDDIR)
-	mkdir -p $@
-
-$(BUILDDIR)/js/tests.js: node_modules $(BUILDDIR)/$(JSON_ACIS) $(BUILDDIR)/js
+$(BUILDDIR)/js/index.html: node_modules $(BUILDDIR)/$(JSON_ACIS) $(BUILDDIR)/$(BYTECODES)
 	npm run browser-test-bundle
 
-browser-tests: $(BUILDDIR)/js/tests.js
+browser-tests: $(BUILDDIR)/js/index.html
 ifneq ($(shell which open),)
-	@open tests/browser/index.html
+	@open build/js/index.html
 else
-	@echo Open "tests/browser/index.html" in your browser.
+	@echo Open "build/js/index.html" in your browser.
 endif
 
+browser-tests-headless: node_modules $(BUILDDIR)/$(JSON_ACIS) $(BUILDDIR)/$(BYTECODES)
+	npx playwright install chromium-headless-shell
```

Comment:
I didn't add this because I thought it could use a local browser installation that is missed in CI. But I found out that `chromium-headless-shell` needs to be installed even locally.

It looks like `npx playwright install` does nothing if a target has already been installed, so I'm not adding extra checks.